pq-crypto/sike_r1/sike_r1_kem.c

```diff
@@ -2,25 +2,26 @@
 * Supersingular Isogeny Key Encapsulation Library
 *
 * Abstract: supersingular isogeny key encapsulation (SIKE) protocol
-*********************************************************************************************/ 
+*********************************************************************************************/
 
 #include "sike_r1_kem.h"
 
 #include <string.h>
 #include "P503_internal_r1.h"
 #include "fips202_r1.h"
 #include "pq-crypto/pq_random.h"
+#include "utils/s2n_safety.h"
 
 int SIKE_P503_r1_crypto_kem_keypair(unsigned char *pk, unsigned char *sk)
 { // SIKE's key generation
   // Outputs: secret key sk (SIKE_P503_R1_SECRET_KEY_BYTES = MSG_BYTES + SECRETKEY_B_BYTES + SIKE_P503_R1_PUBLIC_KEY_BYTES bytes)
   //          public key pk (SIKE_P503_R1_PUBLIC_KEY_BYTES bytes)
 
     digit_t _sk[SECRETKEY_B_BYTES/sizeof(digit_t)];
-    
+
     // Generate lower portion of secret key sk <- s||SK
-    get_random_bytes(sk, MSG_BYTES);
-    random_mod_order_B((unsigned char*)_sk);
+    GUARD_AS_POSIX(get_random_bytes(sk, MSG_BYTES));
+    GUARD(random_mod_order_B((unsigned char*)_sk));
```

Comment:
GUARD_AS_POSIX?

Reply:
It returns `int` for now so a POSIX -> POSIX guard is all we need ATM